### PR TITLE
feat(unlock-protocol-com) - add new products to main menu

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description

**Desktop** 

Screenshot 2023-05-09 at 10 04 50

**Mobile**

Screenshot 2023-05-09 at 10 05 27

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-protocol-com/src/components/layout/DefaultLayout.tsx [hunk 1]
```diff
@@ -132,12 +132,24 @@ const MENU = {
           description: 'Deploy your membership contract with few clicks.',
           url: `${unlockConfig.appURL}/locks`,
         },
+        {
+          title: 'ALPHAtweet',
+          src: '/images/marketing/img-aphatweet.png',
+          url: 'https://alphatweet.xyz/',
+          description: 'Micro-blogging with a twist of referral.',
```
Comment (searchableguy) [inline unlock-protocol-com/src/components/layout/DefaultLayout.tsx:139]: @julien51 is this right description?

Comment (julien51) [reply unlock-protocol-com/src/components/layout/DefaultLayout.tsx:139]: That's for @ccarfi  our wording wizzard!

File: unlock-protocol-com/public/images/marketing/img-aphatweet.png
Comment (ccarfi) [inline unlock-protocol-com/public/images/marketing/img-aphatweet.png:1]: @kalidiagne possible to rename this file and adjust the image tags? filename has a typo in it and should be "img-alphatweet.png" (the current filename is missing the "L" in "alphatweet"

File: unlock-protocol-com/src/components/layout/DefaultLayout.tsx (hunk 1 above)
Comment (ccarfi) [inline unlock-protocol-com/src/components/layout/DefaultLayout.tsx:139]:
```suggestion
          description: 'Make money through creating or sharing content.',
```

File: unlock-protocol-com/src/components/layout/DefaultLayout.tsx [hunk 2]
```diff
@@ -132,12 +132,24 @@ const MENU = {
...
           title: 'Events By Unlock',
           src: '/images/marketing/events.png',
           url: 'https://events.unlock-protocol.com/',
           description: 'Ticketing events with Unlock is simple.',
         },
+        {
+          title: 'Certifications By Unlock',
+          src: '/images/marketing/img-unlock-certification.png',
+          url: 'https://certifications.unlock-protocol.com/',
+          description: 'Certify & show the expertise on chain.',
```
Comment (ccarfi) [inline unlock-protocol-com/src/components/layout/DefaultLayout.tsx:151]:
```suggestion
          description: 'Certify and show expertise on chain.',
```

File: unlock-protocol-com/src/components/layout/DefaultLayout.tsx [hunk 3]
```diff
@@ -132,12 +132,24 @@ const MENU = {
...
           url: `${unlockConfig.appURL}/locks`,
         },
+        {
+          title: 'ALPHAtweet',
+          src: '/images/marketing/img-aphatweet.png',
+          url: 'https://alphatweet.xyz/',
+          description: 'Make money through creating or sharing content.',
+        },
         {
           title: 'Events By Unlock',
```
Comment (ccarfi) [inline unlock-protocol-com/src/components/layout/DefaultLayout.tsx:142]:
```suggestion
          title: 'Events by Unlock Labs',
```

File: unlock-protocol-com/src/components/layout/DefaultLayout.tsx [hunk 4]
```diff
@@ -133,11 +133,23 @@ const MENU = {
...
+          description: 'Make money through creating or sharing content.',
+        },
+        {
+          title: 'Events by Unlock Labs',
           src: '/images/marketing/events.png',
           url: 'https://events.unlock-protocol.com/',
           description: 'Ticketing events with Unlock is simple.',
         },
+        {
+          title: 'Certifications By Unlock',
```
Comment (ccarfi) [inline unlock-protocol-com/src/components/layout/DefaultLayout.tsx:148]:
```suggestion
          title: 'Certification by Unlock Labs',
```

File: unlock-protocol-com/public/images/marketing/img-aphatweet.png
Comment (kalidiagne) [reply unlock-protocol-com/public/images/marketing/img-aphatweet.png:1]: thk! file renamed!

**Comment by kalidiagne** [comment]
Description udpated with screens, and `alphatweet` file  renamed, thk @ccarfi @julien51 !

**Review by julien51 — APPROVED** [review]

**Pull request merged**